java/src/jmri/managers/DefaultShutDownManager.java

```diff
@@ -185,8 +185,13 @@ private boolean runShutDownTasks(boolean isParallel) {
                         log.info("Program termination aborted by \"{}\"", task.getName());
                         return false;  // abort early
                     }
-                } catch (Throwable e) {
+                } catch (Exception e) {
                     log.error("Error during processing of ShutDownTask \"{}\"", task.getName(), e);
+                } catch (Throwable e) {
+                    log.error("Unrecoverable error during processing of ShutDownTask \"{}\"", task.getName(), e);
+                    log.error("Terminating abnormally");
+                    // forcable halt
+                    System.exit(1);
```

Comment:
Do we know that `log.error(..)` will complete before the `System.exit(..)`? I forget what the contract is there.

If not, would it make sense to also send this directly to `System.err`, so we're more likely to get the traceback?

Thanks for considering.